<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="viewGame">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-9</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStand">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblScorePlayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBalance">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance: $1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBet">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bet: $10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBetMinus">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBetPlus">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeal">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DEAL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
